2-collectors/scala-stream-collector/project/Dependencies.scala

```diff
@@ -33,7 +33,7 @@ object Dependencies {
     val awsSdk           = "1.6.10"
     val yodaTime         = "2.1"
     val yodaConvert      = "1.2"
-    val kafka            = "0.10.1.0"
+    val kafka            = "0.10.2.0"
```

Comment:
What I mean is that there is a ticket dedicated to the upgrade and it's separate from the tickets you're trying to solve (e.g. #3325).

Adding the security feature is a separate concern from updating the library.